homeassistant/components/light/yeelight.py

```diff
@@ -85,7 +84,7 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
             device = {'name': device_config[CONF_NAME], 'ipaddr': ipaddr}
             lights.append(YeelightLight(device, device_config))
 
-    add_devices(lights, True)  # true to request an update before adding.
+    add_devices(lights, True)
```

Comment:
This is there to remind that add_devices() has a magic boolean flag. It may also be useful for developers looking how others have done adding. In any case I think this kind of cleanups would deserve their own PR & discussion.